lib/faker/geohash.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Faker.Geohash do
+    import Faker.Address, only: [latitude: 0, longitude: 0]
+    alias Faker.Address
+
+    @moduledoc """
+    Function for generate geohashes.
+    """
+    @doc """
+    Return random geohash.
+    """
+    @spec geohash() :: String.t
+    def geohash do
```

Comment:
I would probably add it as `Faker.Address.geohash/0`. And add a mention in module doc and function doc that you need an optional dependency.

Reply:
I agree